### ISSUE TITLE
Additional Translations

### ISSUE THREAD

**Issue by Flamifly** [FrozenAssassine/EasePass#133]
Fixed some wrong terms for the Translations and added missing once
https://github.com/Flamifly/EasePass/tree/translations

**Comment by Flamifly** [comment]
I saw you changed the way PasswordSafetyChart.xaml will be translated there is a wrong term
PW_SafetyChart_Short_LowerCase
should be 
PW_SafetyChart_Short_LowerCaseLetters

**Comment by FrozenAssassine** [comment]
Oh wait @finn-freitag changed the way the PasswordSafetyChart handles translations, so now all translations are handled in the xaml file directly. I really think this is the better approach. I hope this does not break your changes too much. I will merge this now and I hope everything works fine with emerging on your end then.

**Comment by Flamifly** [comment]
All good there was just some wrong terms at this point

**Comment by FrozenAssassine** [comment]
Yeah, this makes sense, I left the password safety chart a bit untouched in the way of translations due to those weird things that happen when translating to German. But I think you are on the right track there. Let me merge the code from the xaml first and then we can check. Do you have a better approach to reworking this, would be awesome. 
Also, @finn-freitag and I, won't touch the translations anymore, to prevent merge issues with your code😄. 
Oh @finn-freitag already merged the code to move the translations, would you double check that you are up to date in the passwordsafetychart.xaml and .xaml.cs file? Sorry, I can also revert if you want?

**Comment by Flamifly** [comment]
I will change my stuff don't worry

**Comment by Flamifly** [comment]
https://github.com/Flamifly/EasePass/tree/translation2
there will be the translations
Not done yet

**Comment by Flamifly** [comment]
should be finished

**Comment by Flamifly** [comment]
https://github.com/FrozenAssassine/EasePass/issues/133#issuecomment-2546604765
I will have a look tomorrow on that

**Comment by Flamifly** [comment]
Your welcome :smile: 
I am gone for today see ya

**Comment by FrozenAssassine** [comment]
I don't think there was a reason for that, at that time @finn-freitag did not properly know how to handle those translations and so we made a quick and dirty way of doing that. There should be no conflicts, because the names of the language strings point out, that they only are for the PasswordSafetyChart.

**Comment by Flamifly** [comment]
I understand now why it was implemented like that..
The first is the negativ Translations
The second is the positiv Translation
The last is the unknown Translation

Does someone have a idea to have a good way of translation that?

**Comment by Flamifly** [comment]
I could create 3 different Sentences for each Sentence which are very similar to the others

**Comment by FrozenAssassine** [comment]
Well, your idea is pretty simple and ngl I really love that😅

**Comment by Flamifly** [comment]
I will change it to that tomorrow
I also added additional Translations in https://github.com/Flamifly/EasePass/tree/translation2

Tomorrow i will translate the InfoMessages too
![image](https://github.com/user-attachments/assets/a892d929-1833-479d-8d7a-bb04bd25d19f)

**Comment by Flamifly** [comment]
How can I translate the Paragraphs btw?
![image](https://github.com/user-attachments/assets/69b1f443-e7ce-4f93-a574-ca5d2ad8ad12)

**Comment by FrozenAssassine** [comment]
Awesome man ❤️

I don't think we have to translate the Paragraphs or what do you think?

BTW I also added a little new feature with the following new language Strings, I did not merge or commit yet, but I just write so we don't forget lol.
I don't think its smart to add them to the language files now, because we could run into merge issues with your changes:
- View_ManageDB_ExportDBNewPW -> ManageDatabasePage.xaml
- View_PasswordPage_Flyout_ExportNewPassword -> PasswordPage.xaml

**Comment by Flamifly** [comment]
I mean the most people understand english so it is not necessary at all but I personally would say it would be good to translate the privacy policy

What does this Feature do?
Is that the Feature to Backup the Database?

**Comment by FrozenAssassine** [comment]
Well you are correct since the app ships in two languages, it would be the best, to fully translate those languages then.
I could try to translate it for you?

The feature is so you can select some items from the password item pane and rightclick on export diff PW, and a dialog will open up so you can export the selected items from the list as a new database with your given password. This is useful when transferring passwords from one device to another or so.

**Comment by Flamifly** [comment]
I can Translate it if you show me how it works on paragraphs

That sounds cool

**Comment by FrozenAssassine** [comment]
I will give you an example see ya✌️

**Comment by FrozenAssassine** [comment]
> How can I translate the Paragraphs btw? ![image](https://private-user-images.githubusercontent.com/187693389/396656312-69b1f443-e7ce-4f93-a574-ca5d2ad8ad12.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0Njg4NjQsIm5iZiI6MTczNDQ2ODU2NCwicGF0aCI6Ii8xODc2OTMzODkvMzk2NjU2MzEyLTY5YjFmNDQzLWU3Y2UtNGY5My1hNTc0LWNhNWQyYWQ4YWQxMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxN1QyMDQ5MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOGZlZmE4MGJkYjI0NjAzZDRkZGRhZGJiZjY4Y2FjODVjODJlOWUwNDIxNjllNjJhZDcxZjc3NjA5OTk2ZDBjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.joGqPzdlgglScPmLOuIrU6hog6ltZA0qq-90lgock1U)

Oh I just found out, that there are already translation x:Uid in the image you send. But they are just not inside the resource files. So it should be no problem to add them without any trouble.

**Comment by Flamifly** [comment]
I will get on track on that Issue on Friday because today and tomorrow I have not enough time

**Comment by Flamifly** [comment]
Better?
![image](https://github.com/user-attachments/assets/bdace962-185c-4072-8dcf-79c44b72f937)

**Comment by FrozenAssassine** [comment]
I actually like it, way simpler than before I guess.
What do you think @finn-freitag?

**Comment by Flamifly** [comment]
Anyone have a good short english translation for the Passwordlength thing?

**Comment by finn-freitag** [comment]
Full Length:
- English: Password length
- German: Passwortlänge

Short:
- English: Length
- German: Länge

**Comment by FrozenAssassine** [comment]
Hmm, maybe we write something like 

"14 > Zeichen"?

**Comment by Flamifly** [comment]
Does this show if the minimum has reached?

**Comment by Flamifly** [comment]
Or Passwordlength: secure / unsecure?

**Comment by FrozenAssassine** [comment]
Oh yeah way better actually

**Comment by finn-freitag** [comment]
You can't really say that a password is secure from a certain length. "Password length: True" should be displayed if the length is greater than or equal to the minimum length entered in the settings.

I would replace true/false or secure/insecure trough meets minimum/doesn't meet minimun.

**Comment by Flamifly** [comment]
I will change it to meets minimum tomorrow and will add german translation

**Comment by Flamifly** [comment]
German:
![image](https://github.com/user-attachments/assets/be9e94c3-6ebc-42dc-986f-ce6ea5026a27)

English;
![image](https://github.com/user-attachments/assets/53b8e2d2-1475-462d-a2da-6e5dc5c552cf)

Would be done with the translations if the translations are good

**Comment by finn-freitag** [comment]
The German word “Passwordlänge” is written together and for “Veröffentlicht” the boolean value is not translated, maybe you can correct that. Otherwise, good work!

**Comment by Flamifly** [comment]
Need to change it quick thanks

**Comment by Flamifly** [comment]
Changed it but need to test it might throw errors because edited it on my phone

**Comment by Flamifly** [comment]
Messed up with the change as I mentioned so had to fix it
- it can be compiled and executed now

German translation now:
![image](https://github.com/user-attachments/assets/5333aa72-71d8-4cc6-b26e-1a0899ef58ec)

**Comment by Flamifly** [comment]
Not translated
![image](https://github.com/user-attachments/assets/ef12a58d-533c-43d6-b34e-c5b425b280f4)
![image](https://github.com/user-attachments/assets/1b445cd9-845a-46a5-aa82-c745ae1bc0d1)

Plugin Dialog

**Comment by Flamifly** [comment]
A Translation is messing for the Export with different Password
I can fix it later or you guys can

**Comment by FrozenAssassine** [comment]
Oh, totally forgot to translate this. I'll do it real quick
